### DISCUSSION TITLE
Többszörös végfelhasználói regisztráció

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#32] [Általános]
Ügyfélkörünkben is viszonylag gyakori, hogy ugyanaz a magánszemély több cégben (pl. munkahelye könyvelőjeként + saját könyvelőirodájával) is a mi programunkkal dolgozik, azaz a kliensprogram azonosító és a magánszemély azonos.
Jelenleg egy magánszemély egy adott kliensprogram azonosítóval csak egyszer tud végfelhasználói regisztrációt végezni, emiatt a több elkülönült helyen, jogcímen, de azonos szoftverrel történő M2M-használat nem lehetséges.

**Comment by TamasPill** [comment]
Első körben az a kérdésem lenne, hogy ez azért jelent problémát, mert a telepített szoftver 2 különböző munkagépen fut és a felhasználói adatok szinkronizációja nem megoldott a 2 szoftverpéldány között?

**Comment by kdenes6500** [comment]
A telepített szoftver két külön munkagépen, rendszerben fut, köztük nincs semmilyen kapcsolat.
Pl. XY Kft.-nek van egy saját alkalmazottja, aki a programot az XY Kft. székhelyén, az ottani rendszeren használja. Ugyanez a személy az otthoni saját gépén mint egyéni vállalkozó is könyvel.
Ebben az esetben nemcsak nincs, de nem is lehet semmilyen szinkronizáció, kapcsolat a két rendszer között.

**Reply by Csonti-Piller** [reply]
A felvázolt esettel kapcsolatban szakmai egyeztetés folyik, a végső döntésről, illetve a megoldásról a döntést követően tájékoztatást adunk ki.
